man/README.md

```diff
@@ -23,11 +24,11 @@ Using this `Dockerfile`, create a Docker image tagged `docker/md2man`:
 
 Once the image is built, run a container using the image with *volumes*:
 
-    docker run -v /<path-to-git-dir>/docker/man:/docs:rw \
+    docker run -v /<path-to-git-dir>/<docker-fork>/man:/docs:rw \
```

Comment:
I like it.  And you were close. I just had to make a small change to the .sh name (had to prefix ./). The following tests out:

docker run -v $(pwd):/man -w /man -i docker/md2man ./md2man-all.sh

I'll refresh the commit in the morning. Cheers, Mike